src/exporter.py

```diff
@@ -103,6 +104,13 @@ def track_task_event(self, event):
 
         labels = {"name": task.name, "hostname": task.hostname}
 
+        if event['type'] == "task-sent":
+            # Remove the prefix which contains Process ID from hostname: gen5321@hostname => hostname
+            # Prometheus suggests it:
+            # "Do not use labels to store dimensions with high cardinality (many different label values)"
+            _, node = nodesplit(event['hostname'])
+            labels['hostname'] = node
```

Comment:
Shouldn't we do this for metrics of all types?

```
celery_task_received_total{hostname="gen1@django-wtf-beat-7798d75d75-b9tk8",name="django_apps.core.github_tasks.index_contributors"} 0.0

celery_task_sent_total{hostname="celery@django-wtf-worker-0",name="django_apps.core.github_tasks.index_contributors"} 0.0

celery_task_failed_total{exception="",hostname="gen1@django-wtf-beat-7798d75d75-b9tk8",name="celery.backend_cleanup"} 0.0
```

Reply:
You're right, didn't notice it

Reply:
@danihodovic have another look please when you have time, I've cleaned up all `hostname` field